client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -261,16 +261,10 @@ const connectComponent = connect(
 		const momentSiteZone = moment().utcOffset( timezoneOffset );
 		let date = rangeOfPeriod( period, momentSiteZone.locale( 'en' ) ).endOf;
 
-		let quantity = 10;
+		let quantity = 30;
 		switch ( period ) {
-			case 'day':
-				quantity = 30;
-				break;
-			case 'month':
-				quantity = 12;
-				break;
-			case 'week':
-				quantity = 13;
+			case 'year':
+				quantity = 10;
```

Comment:
Kind of to your point above about performance - I figured we can keep the switch here in case we need to roll this back.